src/script.tc.php

```diff
@@ -10,40 +10,23 @@
 // No direct access to this file
 defined('_JEXEC') or die('Restricted access');
 
-$tjInstallerPath = JPATH_ROOT . '/administrator/manifests/packages/tc/tjinstaller.php';
-
-if (File::exists(__DIR__ . '/tjinstaller.php'))
-{
-	include_once __DIR__ . '/tjinstaller.php';
-}
-elseif (File::exists($tjInstallerPath))
-{
-	include_once $tjInstallerPath;
-}
-
 /**
  * Script file of activitystream component
  *
  * @since  0.0.1
  */
-class Com_TcInstallerScript extends TJInstaller
+class Com_TcInstallerScript
```

Comment:
can we not make it a package and install these 2?

Reply:
Why create a package for only one plugin? We can install the plugin on post-flight and also uninstall it on component uninstallation.